rfc6962-bis.xml

```diff
@@ -300,7 +300,7 @@ d0 d1   d2 d3           d0 d1   d2 d3  d4 d5</artwork>
     </section>
     <section title="Submitters">
       <t>
-        Submitters submit certificates or preannouncements of certificates prior to issuance (precertificates) to logs for public auditing, as described below. In order to enable attribution of each logged certificate or precertificate to its issuer, each submission MUST be accompanied by all additional certificates required to verify the chain up to an accepted trust anchor. The trust anchor (a root or intermediate CA certificate) MAY be omitted from the submission.
+        Submitters submit certificates or pre-registrations of certificates prior to issuance (precertificates) to logs for public auditing, as described below. In order to enable attribution of each logged certificate or precertificate to its issuer, each submission MUST be accompanied by all additional certificates required to verify the chain up to an accepted trust anchor. The trust anchor (a root or intermediate CA certificate) MAY be omitted from the submission.
```

Comment:
I prefer the term "preannouncements".  We don't talk about submission equating to any sort of "registration" anywhere else in the document.

http://www.thefreedictionary.com/preannounce "to announce in advance"
http://www.thefreedictionary.com/announce "To make known publicly"

So a preannouncement makes a thing known publicly in advance.  That's exactly what a precertificate does, isn't it?